servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -1,7 +1,7 @@
 [[kiali-tutorial]]
 = Kiali tutorial
 
-Kiali works with Istio to visualize your service mesh topology to provide visibility into features like circuit breakers, request rates, and more.  Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.  Kiali provides an interactive graph view of your namespace in real time.  It can display the interactions at several levels (applications, versions, workloads) with contextual information and charts on the selected graph node or edge.
+Kiali works with Istio to visualize your {ProductShortName} topology to provide visibility into features like circuit breakers, request rates, and more.  Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.  Kiali provides an interactive graph view of your namespace in real time.  It can display the interactions at several levels (applications, versions, workloads) with contextual information and charts on the selected graph node or edge.
```

Comment:
```suggestion
Kiali works with Istio to visualize your {ProductShortName} topology to provide visibility into features like circuit breakers, request rates, and more. Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads. Kiali provides an interactive graph view of your namespace in real time. It can display the interactions at several levels, including Applications, Versions, and Workloads, with contextual information and charts on the selected graph node or edge.
```